SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj

```diff
@@ -11,8 +11,12 @@
 		80C6FD6926BD8AFF00B73E76 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 80C6FD6826BD8AFF00B73E76 /* ContentView.swift */; };
 		80C6FD6B26BD8B0000B73E76 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 80C6FD6A26BD8B0000B73E76 /* Assets.xcassets */; };
 		80C6FD6E26BD8B0000B73E76 /* Preview Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 80C6FD6D26BD8B0000B73E76 /* Preview Assets.xcassets */; };
-		BC8A981527EB6BC900F6A4F2 /* PayPalWebCheckout in Frameworks */ = {isa = PBXBuildFile; productRef = BC8A981427EB6BC900F6A4F2 /* PayPalWebCheckout */; };
 		CB1AC3BE2982BE250081AED6 /* CardPayments in Frameworks */ = {isa = PBXBuildFile; productRef = CB1AC3BD2982BE250081AED6 /* CardPayments */; };
+		CBDEEA182989782D00A460A6 /* FraudProtection in Frameworks */ = {isa = PBXBuildFile; productRef = CBDEEA172989782D00A460A6 /* FraudProtection */; };
+		CBDEEA1A2989782D00A460A6 /* PaymentButtons in Frameworks */ = {isa = PBXBuildFile; productRef = CBDEEA192989782D00A460A6 /* PaymentButtons */; };
+		CBDEEA1C2989782D00A460A6 /* PaymentsCore in Frameworks */ = {isa = PBXBuildFile; productRef = CBDEEA1B2989782D00A460A6 /* PaymentsCore */; };
```

Comment:
[Looks like](https://paypal.sharepoint.com/:w:/r/sites/BTSDK/_layouts/15/Doc.aspx?sourcedoc=%7B490D7E50-64F1-4349-A778-0F5F1725F4AD%7D&file=Northstar%20Module%20Rename.docx&action=default&mobileredirect=true&cid=af03fdf8-1ebc-445b-add5-fd3c5c19e9a1) the last module to rename is `PaymentsCore` --> `CorePayments` to match the others